nat-lab/tests/telio.py

```diff
@@ -363,6 +363,7 @@ def __init__(
         node: Node,
         adapter_type: AdapterType = AdapterType.Default,
         telio_features: TelioFeatures = TelioFeatures(),
+        ip_stack: IPStack = IPStack.IPv4,
```

Comment:
I think it would be much cleaner if client would just use its `node` IP stack.

Reply:
Fixed